web/src/engine/websites/SweetTimeScan.ts

```diff
@@ -0,0 +1,22 @@
+import { Tags } from '../Tags';
+import icon from './SweetTimeScan.webp';
+import { DecoratableMangaScraper } from './../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as Madara from './decorators/WordPressMadara';
+
+@Madara.MangaCSS(/^https?:\/\/sweetscan\.net\/manga\//, 'div.post-title h1')
```

Comment:
The regex will match links that are not mangas [`https://sweetscan.net/manga/a-good-day-to-be-a-dog/cap-70/`](https://sweetscan.net/manga/a-good-day-to-be-a-dog/cap-70/)